components/textbox/textbox_links.jsx

```diff
@@ -6,18 +6,14 @@ import React from 'react';
 import {FormattedMessage} from 'react-intl';
 import {Link} from 'react-router-dom';
 
-import Constants from 'utils/constants';
-import * as Utils from 'utils/utils.jsx';
-
-const PreReleaseFeatures = Constants.PRE_RELEASE_FEATURES;
-
 export default class TextboxLinks extends React.Component {
     static propTypes = {
         showPreview: PropTypes.bool,
         characterLimit: PropTypes.number.isRequired,
         previewMessageLink: PropTypes.string,
         updatePreview: PropTypes.func,
         message: PropTypes.string.isRequired,
+        isMarkDownPreviewEnabled: PropTypes.bool,
```

Comment:
This prop still needs to be renamed to match what's passed in from the `mapStateToProps`. You might also want to mark this with `.isRequired` since it's passed in by the `mapStateToProps` and it would be weird if it was missing.